tools/cli/internal/cli/changelog/metadata/create.go

```diff
@@ -50,7 +50,7 @@ func (o *Opts) Run() error {
 		return nil
 	}
 
-	return openapi.SaveToFile(o.outputPath, openapi.JSON, metadataBytes, o.fs)
+	return openapi.SaveToFile(o.outputPath, openapi.JSON, *o.newMetadata(), o.fs)
```

Comment:
Wouldn't it make more sense to create a variable `metaData` on line 43.
Move the serialization from line 43 into the branch for o.outputPath and return metadata in the save to file?

Like this:
```go
func (o *Opts) Run() error {
	if o.runDate == "" {
		o.runDate = time.Now().Format("2006-01-02")
	}
	
	metadata := *o.newMetadata()

	if o.outputPath == "" {
		metadataBytes, err := json.MarshalIndent(metadata, "", "  ")
		if err != nil {
			return err
		}

		fmt.Println(string(metadataBytes))
		return nil
	}

	return openapi.SaveToFile(o.outputPath, openapi.JSON, metadata, o.fs)
}
```

Reply:
it does 👍 thanks for the feedback! updated.